Batteries/Tactic/Basic.lean

```diff
@@ -0,0 +1,7 @@
+import Lean.Elab.Tactic.ElabTerm
+import Batteries.Linter
+import Batteries.Tactic.Init
+import Batteries.Tactic.SeqFocus
+import Batteries.Util.ProofWanted
+
+-- This is an import Batteries
```

Comment:
is this copypasta?

Reply:
Oh dear, it appears to be a sed gone bad. :-(